AnagramSolver.BusinessLogic/Classes/AnagramSolver.cs

```diff
@@ -0,0 +1,48 @@
+﻿using AnagramSolver.Contracts.Interfaces;
+using AnagramSolver.Models.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AnagramSolver.BusinessLogic.Classes
+{
+    public class AnagramSolver : IAnagramSolver
+    {
+        WordRepository word = new WordRepository();
```

Comment:
Naming _WordRepository_ `word` here could be misleading. You would guess this variable would represent a string or a class called Word. Calling method `GetAllWords()` on variable `word` seems also wrong.
Consider changing variable's name.

Reply:
changed to wordRepository, since it's only used to get one method